custom_helpers.go

```diff
@@ -1,5 +1,22 @@
 package gotgbot
 
+import (
+	"fmt"
+	"strings"
+)
+
+/* GetLink is a helper method to easily get the message link excluding group and private chat types.
+(It will return en empty string in case of private or group chat type)*/
```

Comment:
ultra nit: 
Golang doc generators tend to expect `//` style comments for function documentation. `/**/` comments are typically only used for in-code explanations of weird logic, or to comment out large blocks when debugging.

Reply:
Oh, I was unaware about that, Sorry